frontend/src/components/Pages/InspectionPage/InspectionOverview.tsx

```diff
@@ -34,6 +35,11 @@ const StyledView = styled.div`
     align-items: flex-start;
 `
 
+const StyledButton = styled(Button)`
+    height: auto;
+    min-height: ${tokens.shape.button.minHeight};
+`
+
```

Comment:
doesn't have to be in this PR but we can probably create a general syle and inport it in multiple files

Reply:
https://github.com/equinor/flotilla/pull/1338
Have done that in this one so will fix it there if this one is approved first